x-pack/plugins/enterprise_search/public/applications/shared/layout/layout.scss

```diff
@@ -7,8 +7,12 @@
 .enterpriseSearchLayout {
   $sideBarWidth: $euiSize * 15;
   $sideBarMobileHeight: $euiSize * 4.75;
+  $consoleHeaderHeight: 98px; // NOTE: Keep an eye on this for changes
+  $pageHeight: calc(100vh - #{$consoleHeaderHeight});
 
   display: block;
+  background-color: $euiColorEmptyShade;
+  height: $pageHeight;
```

Comment:
We need to make this `min-height` instead of `height` I think, that way we don't run into issues where the container 'stops' because the content is taller than the viewport height:

<img width="1435" alt="Screen Shot 2020-09-15 at 5 20 39 PM" src="https://user-images.githubusercontent.com/549407/93277875-e5651f80-f777-11ea-86a9-02f45a2eb925.png">

```suggestion
  min-height: $pageHeight;
```